xbmc/guilib/GUISliderControl.cpp

```diff
@@ -218,10 +218,19 @@ bool CGUISliderControl::ProcessSelector(CGUITexture* background,
 
 void CGUISliderControl::Render()
 {
-  if (!IsDisabled())
-    m_guiBackground->Render();
-  else
-    m_guiBackgroundDisabled->Render();
+  const bool renderFrontToBack = CServiceBroker::GetWinSystem()->GetGfxContext().GetRenderOrder() ==
+                                 RENDER_ORDER_FRONT_TO_BACK;
+
+  auto renderBackground = [this]
```

Comment:
I'm pretty sure this falls apart if the foreground object is opaque, while the background object has transparency. The background would render on top, as both occupy the same layer. 

For this case, each UI elements has two layers. See https://github.com/sarbes/xbmc/blob/51bef9501591cc8a753c91f97b8f554ad73049cb/xbmc/guilib/GUIButtonControl.cpp#L111

Reply:
Thanks, the ability to control the ordering was what exactly what I was looking for :slightly_smiling_face: